shotover-proxy/src/transforms/redis/sink_cluster.rs

```diff
@@ -141,7 +141,7 @@ impl RedisSinkCluster {
             // Return an error as we cant send anything if there are no channels.
             0 => {
                 let (one_tx, one_rx) = immediate_responder();
-                match self.connection_error {
+                match self.reason_for_no_nodes {
```

Comment:
unreachable_cluster_error ?

Not to bikeshed on names, but i feel like the name error should be in here?

Reply:
The reason I dont want error in the name is because it makes it sound like None means there is an error and Some("") means there is an error.
Which isnt true.
The variable does not decide if there is an error, it is just used to enhance the error message if available.

But ... maybe this isnt how other people see this at all.
So im happy for `unreachable_cluster_error` if you think that makes more sense.

Reply:
Ahh yup gotcha